### PR TITLE
[Ps/api training] Upload and Inference notebooks added

### PR DESCRIPTION
**Pull request by minhajul-clarifai**
## Improvements
- Added new directory structure
- Improved readme tables
- Added notebooks for uploading
- Added notebooks for inference

## TODO
- Update colab badge links to point to main branch

### CODE REVIEW

**Comment by Scott-Clone** [comment]
Great work on this Minh!

Looks fine to me 👍🏾 

A small thing, we could add the new Clarifai TM logo since this is being used by the community :)

**Review by Scott-Clone — COMMENTED** [review]
Great work! 

All clear on my end I left a small comment for adding new logo :)

**Review by Scott-Clone — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,9 +15,26 @@ Clarifai provides a platform for data scientists, developers, researchers and en
...
+
+
+## Inference
+| Notebook | Open in Colab |Resources |
+| ----------- | ----------- | ----------- |
+| [Basic Inference](https://github.com/Clarifai/colab-notebooks/blob/ps/api-training/inference/basic_inference.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Clarifai/colab-notebooks/blob/ps/api-training/inference/basic_inference.ipynb) |[Clarifai Docs](https://docs.clarifai.com/api-guide/predict/images)|
+| [DataFrame Inference](https://github.com/Clarifai/colab-notebooks/blob/ps/api-training/inference/dataframe_inference.ipynb) | [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Clarifai/colab-notebooks/blob/ps/api-training/inference/dataframe_inference.ipynb) |[Clarifai Docs](https://docs.clarifai.com/api-guide/predict/images)|
```
Comment (chungisaac) [inline README.md:35]: Just wondering why the ilnks have `ps/api-training/inference` and the folder structure here (under `files` of the PR) doesn't?

Comment (minhajul-clarifai) [reply README.md:35]: That is the branch name `ps/api-training`. The TODO is to change that to point to master/main branch after merge

**Review by Scott-Clone — APPROVED** [review]

**Pull request merged**